be/src/storage/rowset/column_reader.cpp

```diff
@@ -83,21 +83,25 @@ ColumnReader::~ColumnReader() {
     if (_ordinal_index_meta != nullptr) {
         MEM_TRACKER_SAFE_RELEASE(GlobalEnv::GetInstance()->ordinal_index_mem_tracker(),
                                  _ordinal_index_meta->SpaceUsedLong());
+        _onetime_meta_bytes.fetch_sub(_ordinal_index_meta->SpaceUsedLong());
```

Comment:
`std::memory_order_relaxed` for clearer memory order semantics

Reply:
done